docs/can-guides/experiment/technology/html.md

```diff
@@ -364,8 +360,8 @@ built within a [Model-View-ViewModel (MVVM) architecture](https://en.wikipedia.o
 <img src="../../docs/can-guides/experiment/technology/can-component.png"
   alt="" class='bit-docs-screenshot'/>
 
-Instead of creating the view, view-model as separate entities, they are
-often done together as follows:
+Instead of creating the view and view-model as separate entities, they are
+often created together as follows:
```

Comment:
I generally try to use `define` instead of `create` to more clearly distinguish between "extending / define" VS "instantiate / create".